client/my-sites/importer/importer-header.jsx

```diff
@@ -84,7 +84,7 @@ class ImporterHeader extends React.PureComponent {
 	};
 
 	getLogo = icon => {
-		if ( includes( [ 'wordpress', 'medium' ], icon ) ) {
+		if ( includes( [ 'wordpress', 'medium', 'blogger-alt' ], icon ) ) {
```

Comment:
Just curious... Why `blogger-alt` & not just `blogger`?

Reply:
Good question! It's using `social-logos` to display the icon and looking at the Blogger one it includes the border, which we don't want:

https://github.com/Automattic/social-logos/blob/master/svg-min-react/blogger.svg

The alt version is just the B logo which is better for styling.